src/libraries/Common/src/System/Security/Cryptography/SlhDsa.cs

```diff
@@ -105,11 +105,8 @@ public void Dispose()
         ///   An optional context-specific value to limit the scope of the signature.
         ///   The default value is an empty buffer.
         /// </param>
-        /// <returns>
-        ///   The number of bytes written to the <paramref name="destination" /> buffer.
-        /// </returns>
         /// <exception cref="ArgumentException">
-        ///   The buffer in <paramref name="destination"/> is too small to hold the signature.
+        ///   The buffer in <paramref name="destination"/> is too incorrect length to receive the signature.
```

Comment:
```suggestion
        ///   The buffer in <paramref name="destination"/> is incorrect length.
```

(note extra `too`)

also consider adding remark to point to the `SlhDsaAlgorithm.SignatureSizeInBytes`